src/main/java/com/iexec/common/precompute/PreComputeUtils.java

```diff
@@ -29,4 +29,7 @@ public class PreComputeUtils {
     public static final String INPUT_FILE_NAMES = "INPUT_FILE_NAMES";
     public static final String IEXEC_PRE_COMPUTE_OUT = "IEXEC_PRE_COMPUTE_OUT";
     // pre-compute writes its output in compute stage's input (/iexec_in).
+
+    private PreComputeUtils() {}
```

Comment:
`@NoArgsConstructor(access = AccessLevel.PRIVATE)`

Reply:
Fixed in fe571ad

Reply:
:pray: